tests/unit/type/stream-cgroups.tcl

```diff
@@ -1257,7 +1257,6 @@ start_server {
             assert {[dict get [r xinfo stream mystream] length] == 0}
             set grpinfo [r xinfo groups mystream]
             r bgrewriteaof
-            waitForBgrewriteaof r
```

Comment:
why was this removed?

Reply:
Hi Oran,
waitForBgrewriteaof  is getting timed out so test getting failed, we could see that loadaof is successful without the wait so omitted it.

Reply:
i don't understand why it would time out, and it does seem necessary for the test (a test that does LOADAOF after a rewrite)